lib/semantics/expression.cc

```diff
@@ -2400,12 +2423,82 @@ void ArgumentAnalyzer::Analyze(
     }
     actuals_.emplace_back(std::move(*actual));
   } else {
-    success_ = false;
+    fatalErrors_ = true;
+  }
+}
+
+bool ArgumentAnalyzer::IsIntrinsicRelational(RelationalOperator opr) {
```

Comment:
Could this (and the following) be `const`?

Reply:
Yes, done